### DISCUSSION TITLE
测试到期之后怎么办？

### DISCUSSION THREAD

**Discussion by Shonel** [Paladinfeng/MiaoProject#161] [General]
请问是不是33天到期之后就不能再用了？
之前没在应用商店购买的是不是就再也用不了了？

**Comment by sliencex** [comment]
同问，需要一个可以购买到途径。

**Reply by codesign2020** [reply]
快到期了，还会开放吗

**Reply by lirundong** [reply]
同担心，这个软件太好用，如果能重新上架商店就好了。不然只能自己学 tvOS 开发，搞一个 yet another miao project

**Reply by hopexn** [reply]
> 同担心，这个软件太好用，如果能重新上架商店就好了。不然只能自己学 tvOS 开发，搞一个 yet another miao project

你太秀了，可以加我一个😂

**Reply by Paladinfeng** [reply]
详情见TG公告 https://t.me/miaoproject
